monash-human-power/MHP_DAS
# Update MQTT topics

## Description

Fairly simple PR. This replaces all MQTT topics in the `das` module with the constants from our `mhp` module, and manually updates the topics in the Ant+ and WM code.

I think it would be possible to swap over the wireless module code to also use common repo topics, but it'll be more complicated so I'll leave that for another day.

This PR depends on monash-human-power/common#12. Once that PR is merged, I'll need to update `pyproject.toml`. Don't approve before this, otherwise I'll totally forget haha (pls still review though)

## Screenshots

n/a

## Steps to Test

Confirm that all scripts work as expected. Keep in mind that if you're testing them with other projects, you'll need to use the updated PRs for them too so that they MQTT topics match!
